docs/discover/context.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[discover-document-context]]
+== View a document in context
```

Comment:
@gchaps maybe we should relabel it to `View surrounding documents`, since that's the name in the interface?

Reply:
View documents before and after an event?

Reply:
This is exactly what it's doing, but it's also new wording , and if e.h. people search for help, they will search for `View surrounding documents`, so it's more likely the will land in this doc page

Reply:
++